collections/concepts/what-are-tokens.md

```diff
@@ -1,32 +1,78 @@
 ---
-layout: post
+layout: post 

```

Comment:
Does the trailing whitespace mean something?

Reply:
No, I autoformatted the file and it messed up this header to where I had to manually add back in newlines - I'll remove these changes